samza-core/src/main/java/org/apache/samza/zk/ZkJobCoordinator.java

```diff
@@ -356,9 +324,8 @@ private JobModel generateNewJobModel(List<String> processors) {
     public void onBecomingLeader() {
       LOG.info("ZkJobCoordinator::onBecomeLeader - I became the leader!");
```

Comment:
Maybe update these messages and remove exclamation marks? Hopefully coordinator working as expected isn't an exciting event anymore :) 

Also, maybe add the processor ID / job model versions to these for context where missing.